src/test/java/junit/StringTest.java

```diff
@@ -0,0 +1,68 @@
+package junit;
+
+import static org.assertj.core.api.Assertions.*;
+import static org.junit.jupiter.api.Assertions.*;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+@DisplayName("String 클래스에 대한 학습 테스트\n")
+class StringTest {
+
+	@DisplayName(" '1,2'를 ,(콤마)로 split 했을 때, 잘 분리되는지 확인하는 테스트")
+	@Test
+	void splitCommaTest() {
+		// given
+		String string = "1,2";
+		// when
+		String[] stringArray = string.split(",");
```

Comment:
```suggestion
		// given
		String string = "1,2";
		
		// when
		String[] stringArray = string.split(",");
```
사소하지만 given, when, then 사이에 라인을 하나 띄워주시면 더욱 가독성이 좋아보이는데요 :) 
취향이니 참고만 해주세요! 😄

Reply:
코멘트 감사합니다 👍